ui/l10n/es.js

```diff
@@ -32,15 +32,15 @@ var dictionary = {
     "error.password.not.match": "Los campos de contraseña no coinciden",
     "error.please.specify.physical.network.tags": "Las Ofertas de Red no están disponibles hasta que se especifique los tags para esta red física.",
     "error.session.expired": "Su sesión ha caducado.",
-    "error.something.went.wrong.please.correct.the.following": "Algo salió mal, por favor corrija lo siguiente",
+    "error.something.went.wrong.please.correct.the.following": "Algo salió mal, por favor, corrija lo siguiente",
     "error.unable.to.reach.management.server": "No es posible alcanzar al Servidor de Gestión",
     "error.unresolved.internet.name": "El nombre de Internet no se puede resolver.",
     "force.delete": "Forzar Borrado",
     "force.delete.domain.warning": "Advertencia: Elegir esta opción, provocará la eliminación de todos los dominios hijos y todas las cuentas asociadas y sus recursos.",
     "force.remove": "Forzar el retiro",
     "force.remove.host.warning": "Advertencia: Elegir esta opción provocará que CloudStack detenga a la fuerza todas las máquinas virtuales antes de eliminar este host del clúster.",
     "force.stop": "Forzar Parar",
-    "force.stop.instance.warning": "Advertencia: Forzar la dertención de esta instancia deberí­a ser su última opción. Puede conducir a la pérdida de datos, así­ como un comportamiento incoherente del estado de la máquina virtual.",
+    "force.stop.instance.warning": "Advertencia: Forzar la dertención de esta instancia debería­a ser su última opción. Puede conducir a la pérdida de datos, así­ como un comportamiento incoherente del estado de la máquina virtual.",
```

Comment:
Extra 'a' on 'debería'

Reply:
It's a weird GitHub bug. Download the file with `wget` and you'll see that the extra 'a' doesn't exist. I didn't report it to GitHub because I don't collaborate with proprietary software.

Reply:
It is because you are writing `debería-a`. For some reason that `-` is being removed in the HTML. It might be some escaping problem.

Reply:
It's a [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen). I couldn't see it with Vim.